About/FAQ.md

```diff
@@ -182,6 +188,16 @@ throttling or limiting the number of failed attempted while TOTP is susceptible
 to phishing attacks and requires a user to enter the code within a given time
 period.
 
+How is HOTP used in heads? TBD
+----
+
+? Wish I knew.
```

Comment:
Can the "? Wish I knew" statements be replaced with "TBD" or "To be written"?  I understand that is your personal perspective but when merged into the official documentation, I do not want to give the impression that no one on the project understands what TOTP and HOTP are included.

Reply:
Done.